### PR TITLE
🔧 drop ended

### LINKED ISSUE #8892: Dynamic indicator on drop cards
image

- the drop ended 

if the drop ended that minted = supply show "Minted out" 
if the drop ended that we ended it, show "Ended"

### PR DESCRIPTION
**Pull request by roiLeo**
- close #8892 

Screenshot 2024-01-06 at 13-17-08 KodaDot - NFT Market Explorer

### CODE REVIEW

File: components/drops/DropCard.vue [hunk 1]
```diff
@@ -108,15 +105,8 @@ const props = defineProps<Props>()
 const image = ref('')
 const externalUrl = ref()
 
-const correctUrlPrefix = computed(() => {
-  return props.drop.chain
-})
```
Comment (Jarsen136) [inline components/drops/DropCard.vue:113]: I prefer to rename this one instead of putting it in several places.

**Review by Jarsen136 — APPROVED** [review]
lgtm

**Comment by prury** [comment]
image

Minted out in this case, no?

**Comment by prury** [comment]
@exezbcz did you meant ended at external card and Minted out inside like the image above? 

image

**Comment by exezbcz** [comment]
@prury could be, for another issue i guess 


I am more interested in why the button is active tho :D 
image

**Comment by prury** [comment]
> @prury could be, for another issue i guess 
> 
> 
> I am more interested in why the button is active tho :D 
> image
> 

When disconnected it stays active all the time, it only gets disabled once you connect your wallet

**Review by preschian — APPROVED** [review]

**Pull request merged**